translations/updateUpstream.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+echo "Attempting to push any changed files."
+
+git config user.name github-actions
+git config user.email github-actions@github.com
+
+echo "$(git status)"
+
+git add "po" "inst/qml/translations" "*.po" "*.pot"
+git diff-index --quiet HEAD
+
+if [ $? = 0 ] ; then
+	echo "No translations files were updated, not comitting anything."
+	exit 0
+fi
+
+# also print the commands for some nicer feedback
+echo "$(git status)"
+echo "$(git commit -m 'updated translation files')"
```

Comment:
Is this commit added to the current PR, or is it a new commit direct in the repository?

Reply:
directly to the repository